src/model/kube/statusFetcher_test.go

```diff
@@ -0,0 +1,30 @@
+package kube
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func TestCheckHealth_AllHealty(t *testing.T) {
+	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Write([]byte("{\"services\": []}"))
+
+	}))
+
```

Comment:
Also close the server

```suggestion
	defer server.Close()
	
```